src/docs/blueprint_download.md

```diff
@@ -46,4 +46,4 @@ The source code for Eclipse Theia Blueprint is available on [Github](https://git
 
 ## Documentation
 
-Please see [here](../blueprint_documentation) for documentation on how to customize Eclipse Theia Blueprint.
+Please see [here](https://theia-ide.org/docs/blueprint_documentation/) for documentation on how to customize Eclipse Theia Blueprint.
```

Comment:
Shouldn't we rather use [here](blueprint_documentation)

Reply:
@JonasHelming the link you proposed does not exist, should we not point to the actual website page and not the source?

Reply:
Sorry, Github interpreted my comment, I meant:

`[here](blueprint_documentation)`

Reply:
@JonasHelming I followed the same format as `summary`, please let me know if it is fine :)

https://github.com/eclipse-theia/theia-website/blame/master/src/docs/summary.md#L29